packages/runtime/plugin-runtime/src/exports/server.ts

```diff
@@ -10,8 +10,17 @@ export type {
   CacheControl,
   CacheOptionProvider,
   CacheOption,
+  UnstableMiddlewareContext,
+  UnstableMiddleware,
```

Comment:
Maybe a different name semantics would be better, because it's not a global middleware.

Reply:
This is not the end state. `UnstableMiddleware` is suit for `Middleware`.
In our design, server would provider a configuration file `server.config.ts`.
We can rename the type in that PR